boards/arm/nucleo_g474re/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+  - 1x low-power timer
+
+- Up to 86 fast I/Os, most 5 V-tolerant
+- Memories
+
+  - Up to 128 KB single bank Flash, proprietary code readout protection
+  - Up to 22 KB of SRAM including 16 KB with hardware parity check
+
+- Rich analog peripherals (independent supply)
+
+  - 2x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200
+    |micro| A/MSPS
+  - 4x 12-bit DAC, low-power sample and hold
+  - 3x operational amplifiers with built-in PGA
+  - 4x ultra-fast rail-to-rail analog comparators
+
+- 16x communication interfaces
+
+  - 1 x FDCAN controller supporting flexible data rate
+  - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 4x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 1x LPUART
+  - 3x SPIs (2x with multiplexed half duplex I2S interface)
+  - 1x SAI (serial audio interface)
+  - USB 2.0 full-speed interface with LPM and BCD support
+  - IRTIM (Infrared interface)
+  - USB Type-C™ /USB power delivery controller (UCPD)
+
+- 12-channel DMA controller
+- True random number generator (RNG)
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell*
+
+
+More information about STM32G474RE can be found here:
+
+- `STM32G474RE on www.st.com`_
+- `STM32G4 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_g474re board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_g474re/nucleo_g474re_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo G474RE Board has 6 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32G4 Nucleo-64 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_1_TX : PC4
```

Comment:
This long list will display as a multi-column list if you add this directive and a blank line before the first list item:
```
.. rst-class:: rst-columns

- UART_1_TX : PC4
- UART_1_RX : PC5
  ...
```

Reply:
ok, could also fit  nucleo_g431rb/